crates/dojo/core/src/world/iworld.cairo

```diff
@@ -273,6 +273,12 @@ pub trait IWorld<T> {
     fn revoke_owner(ref self: T, resource: felt252, address: ContractAddress);
 
 
+    /// Get the number of owners for a resource.
+    ///
+    /// # Arguments
+    ///   * `resource` - The selector of the resource.
+    fn get_number_of_owners(self: @T, resource: felt252) -> u64;
```

Comment:
nitpicking: how `owners_count` sounds to you? Feels like this one is a bit long and can be expressed similarly with `owners_count`. WDYT?

Reply:
Yes, why not 👍 Let's go for `owners_count`!